nbt-extra-kotlin/src/main/kotlin/net/kyori/adventure/nbt/extra/kotlin/BinaryTagConverters.kt

```diff
@@ -0,0 +1,32 @@
+package net.kyori.adventure.nbt.extra.kotlin
+
+import net.kyori.adventure.nbt.ByteArrayBinaryTag
+import net.kyori.adventure.nbt.ByteBinaryTag
+import net.kyori.adventure.nbt.DoubleBinaryTag
+import net.kyori.adventure.nbt.FloatBinaryTag
+import net.kyori.adventure.nbt.IntArrayBinaryTag
+import net.kyori.adventure.nbt.IntBinaryTag
+import net.kyori.adventure.nbt.LongArrayBinaryTag
+import net.kyori.adventure.nbt.LongBinaryTag
+import net.kyori.adventure.nbt.ShortBinaryTag
+import net.kyori.adventure.nbt.StringBinaryTag
+
+public fun Byte.toBinaryTag(): ByteBinaryTag = ByteBinaryTag.of(this)
+
+public fun Short.toBinaryTag(): ShortBinaryTag = ShortBinaryTag.of(this)
+
+public fun Int.toBinaryTag(): IntBinaryTag = IntBinaryTag.of(this)
+
+public fun Long.toBinaryTag(): LongBinaryTag = LongBinaryTag.of(this)
+
+public fun Float.toBinaryTag(): FloatBinaryTag = FloatBinaryTag.of(this)
+
+public fun Double.toBinaryTag(): DoubleBinaryTag = DoubleBinaryTag.of(this)
+
+public fun ByteArray.toBinaryTag(): ByteArrayBinaryTag = ByteArrayBinaryTag.of(*this)
+
+public fun IntArray.toBinaryTag(): IntArrayBinaryTag = IntArrayBinaryTag.of(*this)
+
+public fun LongArray.toBinaryTag(): LongArrayBinaryTag = LongArrayBinaryTag.of(*this)
```

Comment:
For non-bots, I am aware of such a flaw, and there is nothing at all that can be done about it,except for some hacks like creating a class in that package to access constructors that take arrays, and refactoring all of the implementation classes to take arrays rather than varargs, which is something that is not being done.

Reply:
@BomBardyGamer Thank you for the feedback. In this case you can ignore the warning by committing a `.lift/config.toml` file with `ignoreRules = [ "SpreadOperator" ]`.  We're on the lookout for how to improve the results and these sorts of feedback are valuable.